docs/plugins/create-a-plugin.md

```diff
@@ -1,27 +1,46 @@
 ---
 id: create-a-plugin
-title: Create a Backstage Plugin
-description: Documentation on How to Create a Backstage Plugin
+title: Create a Frontend Plugin
+description: Documentation on How to Create a Backstage Frontend Plugin
 ---
 
-A Backstage Plugin adds functionality to Backstage.
+A Backstage Frontend Plugin adds functionality, via extensions, to the Backstage frontend web application.
```

Comment:
Hmm, there's no extensions really in the old system, so not sure that this is true.